api/tests/unit/integrations/github/test_unit_github_views.py

```diff
@@ -675,9 +674,34 @@ def test_github_webhook_delete_installation(
     assert not GithubConfiguration.objects.filter(installation_id=1234567).exists()
 
 
-def test_github_webhook_with_non_existing_installation(
+def test_github_webhook_merged_a_pull_request(
     api_client: APIClient,
+    feature: Feature,
     github_configuration: GithubConfiguration,
+    github_repository: GithubRepository,
+    feature_external_resource: FeatureExternalResource,
+) -> None:
+    # Given
+    settings.GITHUB_WEBHOOK_SECRET = WEBHOOK_SECRET
```

Comment:
If `settings` is being set like this it is better pulling it into the test as a fixture.

Reply:
Added